classes/phing/tasks/ext/phpunit/FormatterElement.php

```diff
@@ -168,6 +168,9 @@ public function getFormatter()
             $this->formatter = new SummaryPHPUnitResultFormatter7($this->parent);
         } elseif ($this->type === "clover") {
             $this->formatter = new CloverPHPUnitResultFormatter7($this->parent);
+        } elseif ($this->type === "clover-html") {
```

Comment:
Should be documented in https://github.com/phingofficial/guide/blob/d1c4b97b72a6ff9cba1801406d871dcfb2aaedcd/source/appendixes/optionaltasks.xml#L8905-L8915